frontend/src/views/ProducerFormPage/NewElementCard.vue

```diff
@@ -0,0 +1,141 @@
+        {{ model.element.name.toLowerCase() }}
+        <span class="uppercase text-gray-400 text-sm ml-2">{{ getType(model.element.objectType) }}</span>
+      </div>
+    </div>
+    <hr class="mt-4 pb-1" />
+    <DsfrInputGroup>
+      <DsfrRadioButtonSet
+        v-model="model.element.authorisationMode"
+        name="authorisationMode"
+        legend="Modalité d'autorisation"
+        :options="authorisationOptions"
+      ></DsfrRadioButtonSet>
+    </DsfrInputGroup>
+    <div v-if="model.element.authorisationMode === 'FR'">
+      <hr class="pb-1 -mt-3" />
+      <div class="">
+        <DsfrInputGroup>
+          <DsfrRadioButtonSet
+            v-model="model.element.frReason"
+            name="frReason"
+            legend="Raison de l'ajout manuel"
+            :options="additionReasons"
+          ></DsfrRadioButtonSet>
+        </DsfrInputGroup>
+        <DsfrInputGroup v-if="model.element.frReason === 'missing'">
+          <DsfrInput
+            v-model="model.frDetails"
+            label-visible
+            label="Information additionnelle"
+            is-textarea
+            hint="Veuillez saissir des informations susceptibles de permettre à l'administration de traiter votre demande"
+          />
+        </DsfrInputGroup>
+      </div>
+    </div>
+    <div v-if="model.element.authorisationMode === 'EU'">
+      <hr class="pb-1 -mt-3" />
+      <div class="grid grid-cols-12 gap-4">
+        <DsfrInputGroup class="col-span-12 md:col-span-3">
+          <DsfrSelect
+            label="Pays de référence"
+            defaultUnselectedText=""
+            v-model="model.referenceCountry"
+            :options="countries"
+            :required="true"
+          />
+        </DsfrInputGroup>
+        <div class="col-span-12 md:col-span-9">
+          <DsfrInputGroup>
+            <DsfrInput
+              v-model="model.euLegalSource"
+              label-visible
+              label="Source réglementaire"
+              is-textarea
+              hint="Veuillez préciser la référence exacte du texte réglementaire."
+            />
+          </DsfrInputGroup>
+          <DsfrInputGroup>
+            <DsfrInput
+              v-model="model.euApplicableRestrictions"
+              label-visible
+              label="Restrictions applicables"
+              is-textarea
+              :required="true"
+              hint="Saisissez ici toutes les informations relatives aux restrictions et qui seraient susceptibles de permettre à l'administration de traiter votre demande. Eviter, si possible, toute information directement ou indirectement nominative."
+            />
+          </DsfrInputGroup>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { getTypeIcon, getType } from "@/utils/mappings"
+const model = defineModel()
+const authorisationOptions = [
+  { label: "🇫🇷 Utilisable en France", value: "FR", hint: "Cet ingrédient est autorisé ou utilisable en France" },
```

Comment:
Pour la petite histoire, Microsoft [n'affiche pas](https://community.sparkleapp.com/t/emoji-flags-not-working-in-edge-browser/4607) les émojis drapeaux dans son browser, pour ne pas se prendre la tête avec les contentieux entre les pays reconnus / non reconnus. Du coup si on veut que l'affichage soit sûr, il faudrait utiliser des icônes plutôt que des emojis.

Reply:
Bien vu ça ! ahah je n'avais pas idée. Je changerai pour des icônes

Reply:
Fix par ici : https://github.com/betagouv/complements-alimentaires/pull/305/commits/12610af419a5a8b0b3540032e8c5c10b9c67e113

Reply:
@alemangui je reviens 2 semaines plus tard 😂...à tout hasard ça marche pas d'utiliser les flag icons (pour pas se casser la tête à ajouter des fichiers) ?

<img width="1306" alt="image" src="https://github.com/betagouv/complements-alimentaires/assets/1933516/cd6b61eb-dc92-4e89-9dde-f2b4c0f7b778">

Reply:
Mieux vaut tard que jamais ^^ En effet je n'ai pas exploré cette option et on a bien un `oh-vue-icons/icons/fi/index.js` qu'on pourrait tenter. J'ai ajouté une petite carte dans le kanban
![image](https://github.com/betagouv/complements-alimentaires/assets/1225929/d71d0cfe-3bfd-41ff-a8e0-45067c84fd77)